k8s/helm/component-patch-tpl.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 values:
   images:
-    snapshotController: registry.k8s.io/sig-storage/snapshot-controller:v5.0.1
+    snapshotController: "{{ .images.snapshotController }}"
```

Comment:
In this section we have to define the raw image refs. No templating here. The refs will be used to template the patches which will then applied to the final helm values. And this process is automated. See `release_args.sh`.